org.civicrm.reqangular/src/common/models/instances/settings-instance.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env amd */
+
+define([
+  'common/modules/models-instances',
+  'common/models/instances/instance'
+], function (instances) {
+  'use strict';
+
+  instances.factory('SettingsInstance', ['ModelInstance',
+    function (ModelInstance) {
```

Comment:
This indentation is a bit difficult to read imho, could you please try that instead:

```js
instances.factory('SettingsInstance', [
  'ModelInstance',
  function (ModelInstance) {
    return ModelInstance.extend({});
  }
]);
```

Reply:
looks better! will implement.